cpu_map_keyme2560.h

```diff
@@ -42,8 +42,13 @@
 #define STEPPERS_DISABLE_PORT  PORTJ
 //  #define STEPPERS_DISABLE_BIT   1 // Atmega2560 pin X / Arduino Digital Pin 14 
 //#define STEPPERS_DISABLE_MASK (0x3C)   //Atmega2560 pins 65-68.   //Arduino xxxx
-#define STEPPERS_DISABLE_MASK (0x3E)   //Atmega2560 pins 65-68.   //temporarily add PJ1, Arduino 14
+#define STEPPERS_DISABLE_MASK (0x3C)   //Atmega2560 pins 65-68.   
 //TODO: separate bits if needed
+
+#define STEPPERS_LONG_LOCK_MASK  (0x10)  //Keep gripper engaged 
```

Comment:
Are we sure this 0x10 mask is correct?  That bit doesn't seem to correspond to anythign with the gripper.  Given the fact that it's in the second nibble, I tried 0xF0 to turn all of the axes long_lock_masks on but that didn't seem to have any effect on the kiosk so I tried 0x0F and that definitely turned things on for the Y table.  Maybe gripper was wrong too?